test/run_test.py

```diff
@@ -573,9 +573,8 @@ def run_test(
 
 def try_set_cpp_stack_traces(env, command, set=True):
     # Print full c++ stack traces during retries
-    if not (len(command) >= 2 and command[2].startswith(INDUCTOR_TEST_PREFIX)):
-        env = env or {}
-        env["TORCH_SHOW_CPP_STACKTRACES"] = "1" if set else "0"
+    env = env or {}
+    env["TORCH_SHOW_CPP_STACKTRACES"] = "1" if set else "0"
```

Comment:
Nit, Don't forget to remove the original import IS_MACOS at the beginning section of the file :)